teams.html

```diff
@@ -15,15 +15,6 @@ <h1>Teams Dashboard</h1>
     <div class="fixed-width">
       <div class="row">
         <ul class="side-navigation">
-          <li class="column l-4-12">
-            <a href="https://studentrobotics.org/ide/">
-              <p class="link-icon"><i class="fa fa-fw fa-2x fa-cogs"></i></p>
-              <p>
-                <span class="link-title">IDE</span>
-                <span class="link-description">Develop and deploy code to your robot.</span>
-              </p>
-            </a>
-          </li>
           <li class="column l-4-12">
```

Comment:
Suggestion: If you change these to `l-4-12` then they should be centered, which may look a bit better